### ISSUE TITLE
FP oelint.vars.dependsappend if using prepend

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#132]
```
DEPENDS_prepend = "abc "
```
gives a false positive
